### PR TITLE
Release v0.11.2

### PR DESCRIPTION
**Pull request by taylortom**
### Issues

- https://github.com/adaptlearning/adapt_authoring/issues/2700
- ~https://github.com/adaptlearning/adapt_authoring/issues/2698~
- https://github.com/adaptlearning/adapt_authoring/issues/2697
- https://github.com/adaptlearning/adapt_authoring/issues/2696
- https://github.com/adaptlearning/adapt_authoring/issues/2693
- https://github.com/adaptlearning/adapt_authoring/issues/2691

### Housekeeping
_To be completed after all PRs merged_
- [x] Update package.json version
- [x] Update package-lock.json
- [x] Update CHANGELOG.md

### CODE REVIEW

File: package.json
Comment (simondate) [inline package.json:1]: Getting 500 errors from any request made.

I imagine this is caused by the upgrade of mongodb from `3.7.3` to `4.10.0`.

I'm testing now reverting these changes.

**Review by simondate — CHANGES_REQUESTED** [review]
Pages aren't loading and getting 500 errors from any API call.

Most likely an issue caused by either the bump of the Node version or the mongodb major version.

File: package.json
Comment (simondate) [reply package.json:1]: I just checked the stable branch that I have on [my own repo](https://github.com/simondate/adapt_authoring/tree/temp/release/0.11.2) and `4.10.0` seemed to be used there.

I'm doing some further testing and will try to find something which works

Comment (simondate) [reply package.json:1]: Code_Nuz2LxLBDP

chrome_PlYzWl5X2Y

**Review by simondate — CHANGES_REQUESTED** [review]

File: package.json
Comment (simondate) [reply package.json:1]: I just google searched the error message and found this forum thread from 3 years ago https://community.adaptlearning.org/mod/forum/discuss.php?d=3811

The issue gets fixed if you run `npm install mongoose@5.8.13` . This version of mongoose was specified in the package.json file so I'm not sure why it wasn't working in the first place. Any ideas @taylortom ?

**Review by simondate — DISMISSED** [review]
Just tested with a clean install with the new package-lock.json file. The couldn't find tennant issue is no longer replicable.

All other issues are still fixed. I've tested #2697 and #2696 and haven't been able to replicate either.

**Comment by taylortom** [comment]
Currently running a production test, will let you know if we see any issues

File: package.json [hunk 1]
```diff
@@ -13,29 +13,29 @@
   },
   "main": "index",
   "engines": {
-    "node": "12 || 14"
```
Comment (hanshillen) [inline package.json:16]: Using node versions higher than 18.15.0 will cause installation of certain plugins to fail, due to https://github.com/ZJONSSON/node-unzipper/issues/271. (after uploading and unzipping, properties.schema will be mangled, causing JSON.parse to fail).

Comment (taylortom) [reply package.json:16]: Have added your comment as an issue (https://github.com/adaptlearning/adapt_authoring/issues/2735) for discussion with the dev team :)

**Comment by taylortom** [comment]
This has been tested in a production environment, with no new issues reported.

**Pull request merged**